psi/pmtdescriptor.go

```diff
@@ -273,3 +280,38 @@ func (descriptor *pmtDescriptor) IsDolbyATMOS() bool {
 	}
 	return false
 }
+
+// Check if the registration data is set to `DOVI`
+// dolby-vision-bitstreams-in-mpeg-2-transport-stream-multiplex-v1.2, Section 3.1
+func (descriptor *pmtDescriptor) IsDolbyVision() bool {
+	if descriptor.tag == REGISTRATION {
+		if len(descriptor.data) >= 4 {
```

Comment:
Minor, strict: From dolby-vision-bitstreams-in-mpeg-2-transport-stream-multiplex-v1.2.pdf, should this be ` 		if len(descriptor.data) == 8 {` ?

Reply:
Tag and length are not included in data. so data should only be 32bits long.